cmake/FindPDAL.cmake

```diff
@@ -7,8 +7,7 @@
 # CMake module to search for PDAL library
 #
 # If it's found it sets PDAL_FOUND to TRUE
-# and following variables are set:
-#    PDAL_INCLUDE_DIR
+# and adds the following target(s):
 #    PDAL_LIBRARIES
```

Comment:
```suggestion
#    - pdalcpp
#    - pdal_util
```

Reply:
The PDAL config-files defines `PDAL_LIBRARIES`, which may be either a list `pdalcpp;pdal_util` (<2.6) or only `pdalcpp` (2.6+).  So PDAL_LIBRARIES should stay, but I may add a clarifying comment.

Reply:
Good point, let's keep as it is then 👍